frontend/src/components/Dashbard.tsx

```diff
@@ -127,14 +130,17 @@ export const Dashboard = (): JSX.Element => {
 		<div>
 			<TopBar/>
 			<div className="outer-container">
-				<MainBreadcrumbs paths={paths}/>
 				{/*Error Message dialogue*/}
 				<ActionModal actionOpen={errorOpen} actionTitle="Something went wrong..." actionText={reason}
 							 actionBtnName="Report" handleActionBtnClick={handleErrorReport}
 							 handleActionCancel={handleErrorCancel}/>
+				<Box m={1} display="flex" justifyContent="space-between" alignItems="flex-end">
+					<MainBreadcrumbs paths={paths}/>
+					<Button href="/create-dataset" variant="contained" sx={{display: "flex", alignItems: "center"}}>New
+						Dataset</Button></Box>
 				<div className="inner-container">
 					<Grid container spacing={4}>
-						<Grid item xs={12} md={8}>
```

Comment:
I think the xs, md, lg, xl might be needed on the grid. otherwise it looks weird with small number of items

Reply:
Thanks for catching this. Should be fixed now